metrics/scaling/k8s_scale_rapid.sh

```diff
@@ -211,8 +211,8 @@ help()
 	usage=$(cat << EOF
 Usage: $0 [-h] [options]
    Description:
-	Launch a series of workloads and take memory metric measurements after
-	each launch.
+	Launch a series of workloads and take memory metric measurements using 'collectd'
+	after each launch.
```

Comment:
In fact, we don't just take memory measurements any more - I think saying `a set of system level measurements using collected` will now be more accurate.